OpenCL_C.txt

```diff
@@ -72,82 +72,141 @@ in OpenCL C.
 Please refer to the C99 specification for a detailed description of the
 language grammar.
 
-The OpenCL C 3.0 programming language includes both required features and
-optional features.
-When an OpenCL C 3.0 optional feature is supported in the language, support
-will be indicated using a __feature test macro__.
+[NOTE]
+--
+The OpenCL 3.0 C programming language described in this specification is provisional.
+It has been Ratified under the Khronos Intellectual Property Framework and is being made publicly available to enable review and feedback from the community.
+
+If you have feedback please create an issue on https://github.com/KhronosGroup/OpenCL-Docs/
+--
```

Comment:
This section has been removed from the latest specifications.

Reply:
I have rebased now, so this section is removed. Thanks!